### PR TITLE
Added zio-json instances for http4s 

### LINKED ISSUE #240: Add Http4s support
We can add integration with Http4s by providing a zio-json `EntityDecoder`/`EntityEncoder` generator

### PR DESCRIPTION
**Pull request by Adriani-Furtado**
closes #240

### CODE REVIEW

**Comment by remiguittaut** [comment]
Looks good to me 😄

File: build.sbt [hunk 1]
```diff
@@ -195,6 +196,24 @@ lazy val zioJsonYaml = project
   )
   .dependsOn(zioJsonJVM)
 
+lazy val zioJsonHttp4s = project
+  .in(file("zio-json-http4s"))
+  .settings(stdSettings("zio-json-http4s"))
```
Comment (fsvehla) [inline build.sbt]: `zio-json-interop-http4s`?

File: zio-json-http4s/src/main/scala/zio/json/http4s/ZIOEntityDecoder.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package zio.json.http4s
```
Comment (fsvehla) [inline zio-json-http4s/src/main/scala/zio/json/http4s/ZIOEntityDecoder.scala]: `zio.json.interop.http4s`

**Comment by fsvehla** [comment]
Thank you for working on this, white wizard.

**Pull request merged**